### PR TITLE
Splitting the visualization of canvas and improve readability of visulization

### LINKED ISSUE #316: AudioWorklet Recorder doesn't display waveform correctly.
1. Go to https://googlechromelabs.github.io/web-audio-samples/audio-worklet/migration/worklet-recorder/
2. Click the page and start recording.
3. Nothing is displayed in the Recording waveform visualization.
4. However, playing the recorded audio works fine.

The screenshot after recording ~5 seconds of audio:
image

### PR DESCRIPTION
**Pull request by TenghuiZhang**
Fix #316 

Splitting the gain value which is used in each visualization canvas.
Multiple the amplitude with a constant number 5 to improve the visualization.

### CODE REVIEW

File: src/audio-worklet/migration/spn-recorder/app.js [hunk 1]
```diff
@@ -237,10 +238,14 @@ function setupVisualizers() {
 
       currentSampleGain /= (currentSamples.length * currentSamples[0].length);
 
-      drawLiveGain(currentSampleGain);
+      const liveGain = currentSampleGain * monitorNode.gain.value;
+      // "Times 5" make the visulization more clear to the users
+      drawLiveGain(liveGain * 5);
```
Comment (hoch) [inline src/audio-worklet/migration/spn-recorder/app.js:243]: I would create `WAVEFROM_SCALE_FACTOR = 5` at the top of this file with a relevant comment.

Do you know the max/min value of `liveGain`? That helps you decide the scale factor accordingly.

File: src/audio-worklet/migration/spn-recorder/app.js [hunk 2]
```diff
@@ -237,10 +238,14 @@ function setupVisualizers() {
...
 
       if (isRecording) {
-        drawRecordingGain(currentSampleGain);
+        const recordGain = currentSampleGain;
+        // "Times 5" make the visulization more clear to the users
+        drawRecordingGain(recordGain * 5);
```
Comment (hoch) [inline src/audio-worklet/migration/spn-recorder/app.js:248]: Ditto.

**Review by hoch — COMMENTED** [review]
Let's think about the right approach of using a scale factor.

File: src/audio-worklet/migration/spn-recorder/app.js (hunk 1 above)
Comment (TenghuiZhang) [reply src/audio-worklet/migration/spn-recorder/app.js]: That's a good point! The canvas height is 150, so I believe the max value of Y is 75. Actually, I choose 5 is based on the manual testing visualization. Based on the calculation, I can not find a good factor. Do you know any formula which might help us to identify a better value for WAVEFROM_SCALE_FACTOR?

**Review by hoch — APPROVED** [review]
LGTM

**Review by hoch — APPROVED** [review]
LGTM

**Pull request merged**